docs/source/azure/configuration/acitutorial.rst

```diff
@@ -63,11 +64,26 @@ storage account created above.
     az role assignment create --assignee <Managed Identity Client Id> \
         --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
 
-4. Create the ACI Container Host
+4. Create an Application Insights Instance
+------------------------------------------
+
+We will point the container host to this Application Insights to gather logs and monitor the host
```

Comment:
"We will create an Application Insights instance to gather the logs and telemetry generated by the running container host"